Source/PLCrashReport.h

```diff
@@ -28,19 +28,83 @@
 
 #import <Foundation/Foundation.h>
 
+#if __has_include(<CrashReporter/PLCrashReportApplicationInfo.h>)
```

Comment:
if this file is umbrella for framework - no need for checking at all. If this used in static libraries - we can make separate file for it